Docs: add Cloud what's new items to 10.2 what's new (#76642)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-2.md

Learn how to upload images here: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets
-->

## Public dashboards

<!-- Thanos Karachalios -->

> _Generally Available in Grafana Enterprise_

Reviewer: Not only on Enterprise, this will be GA in OSS as well.

> _Generally Available in Grafana Enterprise_

Author: ```suggestion
_Generally Available in Grafana Open Source and Enterprise_
```

This is not only useful in the stat visualization; gauge, bar gauge, and status history visualizations, table cells formatted by thresholds, and gauge table cells all benefit from this addition.

> ### New browse dashboards

Reviewer 2: Copied across the latest changes for new Browse Dashboards https://github.com/grafana/grafana/pull/76567 👍🏻

TODO: Add image / gif / video (maybe not for this one)

## Data sources

### Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute

<!-- Joey Tawadrous, Jen Villa -->

_Experimental in Grafana Open Source and Enterprise_

(Requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater)

> We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.

Reviewer 3: ```suggestion
We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Tempo's [metrics summary API](https://grafana.com/docs/tempo/latest/api_docs/metrics-summary/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```

> This feature is disabled by default. To enable it, file contact Grafana Support.

Reviewer 3: ```suggestion
This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/next/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).
```

> This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/next/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).

Reviewer 3: @imatwawana -- note that my link has a "next" in there. I'm not sure that's correct.

The Datadog data source supports blocking API requests based on upstream rate limits (for metric queries). With this update, you can set a rate limit percentage at which the plugin stops sending queries.

To learn more, refer to [Datadog data source settings](/docs/plugins/grafana-datadog-datasource/latest#configure-the-data-source), as well as the following video demo.

{{< video-embed src="/media/docs/datadog/datadog-rate-limit.mp4" >}}

### Query-type template variables for Tempo data source

<!-- Fabrizio Casati -->

_Generally available in Grafana Open Source and Enterprise_

The Tempo data source now supports query-type template variables. With this update, you can create variables for which the values are a list of attribute names or attribute values seen on spans received by Tempo.

To learn more, refer to the following video demo, as well as the [Grafana Variables documentation](/docs/grafana/next/dashboards/variables/).

{{< video-embed src="/media/docs/tempo/screen-recording-grafana-10.2-tempo-query-type-template-variables.mp4" >}}

## Explore

### Content outline

<!-- Thanos Karachalios -->

_Generally Available in Grafana Open Source and Enterprise_

> Introducing Content Outline in Grafana **Explore**. We recognized that complex mixed queries, as well was lengthy logs and traces results led to to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.

Reviewer 4: ```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized that complex mixed queries, as well as lengthy logs and traces results led to to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```

Author: Thanks for the catch!

> Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, as well as, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.

Reviewer 4: ```suggestion
Introducing Content Outline in Grafana **Explore**. We recognized the challenges of complex mixed queries, lengthy logs and traces results, leading to time-consuming navigation and the loss of context. Content outline is our first step towards seamless navigation from log lines to traces and back to queries ensuring quicker searches while preserving context. Experience efficient, contextual investigations with this update in Grafana Explore. To learn more, refer to the [Content outline documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/#content-outline), as well as the following video demo.
```
